probert/multipath.py

```diff
@@ -42,6 +43,9 @@ def _extract_mpath_data(cmd, show_verb):
     for line in data.splitlines():
         mp_dict = None
         try:
+            field_vals = line.split()
+            log.debug('Extracted multipath %s fields: %s',
+                      show_verb, field_vals)
             mp_dict = mptype(*line.split())._asdict()
```

Comment:
Should we do `*field_vals` here to avoid splitting twice?

Reply:
Yes

Reply:
This change ended up on the wrong line, I think.

Reply:
Oh, I see. Yes